client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -338,9 +350,15 @@ const mapStateToProps = ( state, { commentsPage, siteId } ) => {
 		}
 	} );
 
+	const dataRequests = get( state, 'dataRequests' );
```

Comment:
note that this is a redundant `get()` because it does the exact same thing as accessing `state.dataRequests` except it adds a function call